bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -2256,6 +2260,11 @@ class BinaryFunction {
   /// Returns the last computed hash value of the function.
   size_t getHash() const { return Hash; }
 
+  /// Returns the function hash from pseudo-probe description of the function.
+  size_t getPseudoProbeDescHash() const { return PseudoProbeDescHash; }
```

Comment:
Is this intentional to use `size_t`? looks all other places use  `uint64_t` for pseudo-probe hash.

Reply:
Good catch.